cocotb/__init__.py

```diff
@@ -95,13 +96,37 @@ def _reopen_stream_with_buffering(stream_name):
 # so that cocotb.scheduler gives you the singleton instance and not the
 # scheduler package
 
-scheduler = None
+scheduler = None  # type: cocotb.scheduler.Scheduler
 """The global scheduler instance."""
 
-regression_manager = None
+regression_manager = None  # type: cocotb.regression.RegressionManager
+"""The global regression manager instance."""
 
-plusargs = {}
-"""A dictionary of "plusargs" handed to the simulation."""
+argv = None  # type: List[str]
+"""The argument list as seen by the simulator"""
+
+argc = None  # type: int
+"""The length of :data:`cocotb.argv`"""
+
+PLUSARGS = None  # type: Dict[str, Union[bool, str]]
+"""A dictionary of "plusargs" handed to the simulation. See :make:var:`PLUSARGS` for details."""
+
+plusargs = PLUSARGS  # preserves legacy naming
+
+LANGUAGE = os.getenv("TOPLEVEL_LANG")  # type: str
+"""The value of :make:var:`TOPLEVEL_LANG`"""
+
+SIM_NAME = None  # type: str
```

Comment:
```suggestion
SIM_NAME = ""  # type: str
```
Would not be better like this?
It just makes it easier to filter later on do not need a check for `None`.
Like here:
https://github.com/cocotb/cocotb/blob/beff946fe43a9a73c4c352b5ca0986308446a41a/tests/test_cases/test_cocotb/test_handle.py#L48

Reply:
If its `None` the tests can't be run anyway, so I don't think that matters.